src/neuroconv/datainterfaces/ecephys/openephys/openephyslegacydatainterface.py

```diff
@@ -0,0 +1,83 @@
+"""Authors: Szonja Weigl, Cody Baker."""
+from typing import Optional, List
+
+from ..baserecordingextractorinterface import BaseRecordingExtractorInterface
+from ....utils import FolderPathType, get_schema_from_method_signature
+
+
+class OpenEphysLegacyRecordingInterface(BaseRecordingExtractorInterface):
+    """Primary data interface for converting legacy Open Ephys data (.continuous files).
+    Uses :py:class:`~spikeinterface.extractors.OpenEphysLegacyRecordingExtractor`."""
+
+    ExtractorName = "OpenEphysLegacyRecordingExtractor"
+
+    @classmethod
+    def get_source_schema(cls):
+        """Compile input schema for the RecordingExtractor."""
+        source_schema = get_schema_from_method_signature(
+            class_method=cls.__init__,
+            exclude=["stream_name", "block_index", "all_annotations"],
+        )
+        source_schema["properties"]["folder_path"][
+            "description"
+        ] = "Path to directory containing OpenEphys legacy files."
+        return source_schema
+
+    def __init__(
+        self,
+        folder_path: FolderPathType,
+        stream_name: Optional[str] = "Signals CH",
```

Comment:
Is this really optional? Can it be None?

Reply:
I believe it can be `None` in SI, indicating to load whatever stream is available if there is only one stream (and will fail with error otherwise)

Reply:
That works too